infra/abbreviations.json

```diff
@@ -0,0 +1,137 @@
+{
+  "analysisServicesServers": "as",
```

Comment:
Shall we remove the abbreviations? Since the simplest sample is preferred.

Reply:
@fangjian0423 what do you think of it? I feel like these abbrs could remain the same for all templates. But since this sample is so simple that it only uses one abbr for ASA, so maybe we can remove the whole abbr file and hardcode the asa name in main.bicep directly?

Reply:
Don't remove them, keep the same with other AZD templates.